frontend/app/controllers/Security.scala

```diff
@@ -56,13 +56,20 @@ object Security extends Controller with LazyLogging {
 
     SentryLogging.ravenOpt.fold {
       logger.error(report.message)
-    } {
-      _.sendEvent(new EventBuilder()
-        .withMessage(report.message)
-        .withLevel(RavenEvent.Level.INFO)
-        .withTag("CSP Directive", report.directiveTag)
-        .build()
-      )
+    } { raven =>
+      if(report.message.nonEmpty) {
```

Comment:
I don't see how the message _can_ be empty - tho' we do see it as empty in Sentry - so I would not expect this `if` condition to stop the weird reports we're seeing in Sentry.